### PR TITLE
update installation documentation to be more user-friendly

### PR DESCRIPTION
**Pull request by grawlinson**
Intended to fix issue #2441 

Let me know if further adjustments are required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,12 +28,28 @@ The website and documentation for Pug are still being updated, but if you are ne
...
+
 via npm:
 
 ```bash
 $ npm install pug
 ```
 
+### Command Line
+
+After installing the latest version of [node](http://nodejs.org/), install with:
```
Comment (TimothyGu) [inline README.md]: `s/node/Node.js/`

File: README.md [hunk 2]
```diff
@@ -28,12 +28,28 @@ The website and documentation for Pug are still being updated, but if you are ne
...
+
+```console
```
Comment (TimothyGu) [inline README.md]: Replace `console` with `bash` please for consistency.

**Comment by TimothyGu** [comment]
Just a few cosmetics while you are at it :)

**Comment by grawlinson** [comment]
Righto, renamed the node references as well as console/bash code blocks.

**Comment by TimothyGu** [comment]
Looks good to me. Thanks!

**Pull request merged**